app/database/schemas.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class NoteSchema(BaseModel):
+    title: str
+    description: str
```

Comment:
From the definiation in Models.py it seems that `description` can be nullable.
If so please cahnge - `description: Optional[str] = None`

Reply:
Thanks, you are correct, I'll change it